command/snapshot/inspect/formatter.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 type Formatter interface {
-	Format(*OutputFormat) (string, error)
+	Format(*OutputFormat, bool) (string, error)
```

Comment:
We could remove passing this bool around by just checking the `OutputFormat` to see if kvStats has stuff in it